skopt/tests/test_searchcv.py

```diff
@@ -42,6 +42,15 @@ def test_searchcv_runs(surrogate, n_jobs):
     # None search space is only supported when only `step` function is used
     assert_raises(ValueError, BayesSearchCV(SVC(), None).fit, (X, y))
 
+    # check if invalid dimensions are raising errors
+    def constructor(x): BayesSearchCV(*x)
+    assert_raises(
+        ValueError, constructor, (SVC(), {'C':'1 ... 100.0'})
+    )
```

Comment:
```python
with pytest.raises(ValueError):
  BayesSearchCV(args_here)
```
https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions for soem more examples